.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
 * @arthurevans
+* @justinfagnani
```

Comment:
Do we actually want codeowners? I'm not sure it's actually too useful. It means you and arthur will get assigned to PR.

Reply:
We've been doing it for Lit repos, and this was already here... I thought it might be mildly useful in a monorepo when we might have different owners for different folders. I don't care so much.